### PR TITLE
4.3

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!tastemetitus

File: scripts/install [hunk 1]
```diff
@@ -165,94 +125,29 @@ datadir=/home/yunohost.app/$app
...
-sudo -u $app env PATH=$PATH drush @$app sql-query "INSERT INTO public.ldap_authorization VALUES (1, 'localhost', 'drupal_role', 'ldap_authorization_drupal_role', 1, 1, 0, 'a:2:{i:0;a:6:{s:12:\"user_entered\";s:24:\"utilisateur authentifié\";s:4:\"from\";s:51:\"cn=framaforms.main,ou=permission,dc=yunohost,dc=org\";s:10:\"normalized\";s:24:\"utilisateur authentifié\";s:10:\"simplified\";s:24:\"utilisateur authentifié\";s:5:\"valid\";b:0;s:13:\"error_message\";s:112:\"Role <em class=\"placeholder\">utilisateur authentifié</em>_name does not exist and role creation is not enabled.\";}i:1;a:6:{s:12:\"user_entered\";s:13:\"administrator\";s:4:\"from\";s:52:\"cn=framaforms.admin,ou=permission,dc=yunohost,dc=org\";s:10:\"normalized\";s:13:\"administrator\";s:10:\"simplified\";s:13:\"administrator\";s:5:\"valid\";b:0;s:13:\"error_message\";s:101:\"Role <em class=\"placeholder\">administrator</em>_name does not exist and role creation is not enabled.\";}}', 1, 0, 1, 1, 1, 1);"
-
-cat ../conf/ldap.conf | sudo -u $app env PATH=$PATH drush @$app variable-set --format=yaml ldap_authentication_conf -
```
Comment (alexAubin) [inline scripts/install:246]: Uuuuuuh, is this stuff really not needed anymore ? o_O

**Comment by ericgaspar** [comment]
The package seems to have changed a lot. The install process looks simpler : https://yakforms.org/pages/install.html

**Comment by alexAubin** [comment]
Alrighty, good news then :+1: :stuck_out_tongue:

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 2]
```diff
@@ -55,6 +56,7 @@ ynh_app_setting_set --app=$app --key=password --value=$password
 ynh_app_setting_set --app=$app --key=language --value=$language
 ynh_app_setting_set --app=$app --key=expiration --value=$expiration
 ynh_app_setting_set --app=$app --key=deletion --value=$deletion
```
Comment (alexAubin) [inline scripts/install:58]: (Changes in the manifest.json to be propagated here and elsewhere)

**Comment by alexAubin** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
Shall we rename this repo Yakforms?

**Comment by Tagadda** [comment]
I guess we should. Does this PR handle migration from framaforms ?

**Comment by ericgaspar** [comment]
> I guess we should. Does this PR handle migration from framaforms ?

No migration considered in this PR. Maybe we could create another repo named Yakforms alongside this one.

**Comment by Tagadda** [comment]
When bitwarden_rs was [renamed to vaultwarden](https://github.com/YunoHost-Apps/vaultwarden_ynh/pull/139), repo was renamed, a command was added to the README for the migration.
For the migration, we could use [this helper](https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_handle_app_migration/ynh_handle_app_migration) in the upgrade script.

**Comment by Tagadda** [comment]
I did try this PR and after install I had to setup drupal in my browser (language, database, site info...). I think there was some stuff in the install script that ends the drupal setup automagically and it has been removed in this PR.
French language pack was missing too.

**Pull request closed without merging**